bentoml/_internal/bento/build_dev_bentoml_whl.py

```diff
@@ -62,6 +62,6 @@ def build_bentoml_editable_wheel(target_path: str) -> None:
             # copy the built wheel file to target directory
             shutil.copytree(dist_dir, target_path)  # type: ignore
     else:
-        logger.info(
+        logger.info(  # pragma: no cover
```

Comment:
why "no cover" here? We can come back and add tests for this, no reason to exclude it